charts/pega/values.yaml

```diff
@@ -227,3 +227,7 @@ hazelcast:
   enabled: false
   # No. of initial members to join
   replicas: 3
```

Comment:
Can we have these fields only in sub-chart . We will just try to keep minimal params in Values.yaml file.